chaise-config-sample.js

```diff
@@ -7,5 +7,14 @@ var chaiseConfig = {
     showBadgeCounts: false,
     recordUiGridEnabled: false,
     recordUiGridExportCSVEnabled: true,
-    recordUiGridExportPDFEnabled: true
+    recordUiGridExportPDFEnabled: true,
+    tour: {
```

Comment:
Not sure if this is the right place to add tour info; we should avoid putting specific, custom things in here. What's this object for and can it be moved elsewhere?